core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -304,6 +304,7 @@ oppia.directive('conversationSkin', [
           $scope.recommendedExplorationSummaries = null;
           $scope.answerIsCorrect = false;
           $scope.conceptCardIsBeingShown = false;
+          $scope.oldConceptCardIsBeingShown = false;
```

Comment:
I don't think I understand this conceptCard/oldConceptCard thing. Isn't there always just one concept card available at any given time? I feel like some documentation or variable name changes are in order.

Reply:
Oh, the main reason for two variables was that, ProgressNavDirective needed to be shown if the concept card was visited by clicking the arrows and returning to it (in which case it's an 'old' concept card), so that the arrows for navigating across old cards are present in the concept card also, but this should not be shown if the concept card being shown was just fetched from the backend. I'll add comments explaining this.

Reply:
Hm, I think this is an example of how treating the concept card as a different "thing" from the existing exploration cards complicates matters...